repose-aggregator/components/filters/ip-classification/src/main/scala/edazdarevic/commons/net/CIDRUtils.java

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
Well, I have some unit tests that verify that some addresses match :D, and I found this guy like 3 times on stack overflow